community-build/src/scala/dotty/communitybuild/projects.scala

```diff
@@ -27,8 +29,18 @@ def exec(projectDir: Path, binary: String, arguments: Seq[String], environment:
   log(command.mkString(" "))
   val builder = new ProcessBuilder(command: _*).directory(projectDir.toFile).inheritIO()
   builder.environment.putAll(environment.asJava)
+  
   val process = builder.start()
   val exitCode = process.waitFor()
+  if (exitCode != 0) {
+    val errorStream = process.getErrorStream()
+    val isReader = new InputStreamReader(process.getErrorStream())
+    val br = new BufferedReader(isReader)
+    Iterator.continually(br.readLine()).takeWhile(_ != null).foreach(error(_))
```

Comment:
Is this just for debugging?  Or is there some error output that is not captured by the current infrastructure?

Reply:
I have missed some output, but I do not think, I need it anymore.